easyprivacy/easyprivacy_specific_international.txt

```diff
@@ -209,6 +209,7 @@
 ||mp.weixin.qq.com/mp/jsmonitor?
 ||mp.weixin.qq.com/mp/report?
 ||mp.weixin.qq.com/mp/webcommreport?
+||msg.qy.net^
```

Comment:
Not that it would be anything but a question.  But from my visit to `qy.net` I'm just getting redirected to `iqiyi.com` meaning.. is there any reasons to **NOT** block this domain by `qy.net`? 

I only sees calls to msg.$domain

Reply:
Just spotted these to answer my own Q :smirk:

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +237 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=30&tm=4.709&p1=1_10_222&s1=1&r=&u=8649f23e12dc63353be5d879896bc9db&rn=RzUSGejY4P&nu=undefined&dfp=a186d13400d4af51fbbbe24e855fbfed34a364b21c23d618633ca21c83207ce059&pu=&pagev=&c1=&tmlpt=&rpage=home&qtcurl=https%3A%2F%2Fwww.iq.com%2F&stime=1631805339028&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A39%20GMT&mod=intl&lang=en_us&purl=https%3A%2F%2Fwww.iq.com%2F&hu=-1&de=1631805339.1631805339.1631805339.1&ce=ef432b68a0a54c81192ad37d1129700a` |
| +234 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?u=8649f23e12dc63353be5d879896bc9db&pu=&rn=156215037&p1=1_10_222&de=1631805339.1631805339.1631805339.1&stime=1631805340621&ce=aafe05811deec7eb410d25cd5c1c48c7&bstp=0&t=22&rpage=home&hu=-1&dfp=a186d13400d4af51fbbbe24e855fbfed34a364b21c23d618633ca21c83207ce059&mod=intl&purl=https%3A%2F%2Fwww.iq.com%2F&nu=1&vfm=&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&pcau=0&ptp=&pagev=homepage_adv_v1&coop=&lrfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&timezone=Thu,%2016%20Sep%202021%2015:15:40%20GMT&lang=en_us&re=1920*1080&s2=&s3=&abtest=undefined&fc=&fv=&b_ext={%22bip%22:%22188.176.49.219%22}` |
| +234 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=0&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=pmy0kmnj5ugqkwpd65fogt02&rn=0.5101726529150857&block=sidenavi_download&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=https%253A%252F%252Fwww.iqiyi.com%252Fv_1en7t3axhk4.html%253Fvfrm%253Dpcw_home%2526vfrmblk%253DO%2526vfrmrst%253D712211_dianshiju_image1&mod=intl&lang=en_us&stime=1631805339047&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A39%20GMT&stype=&dfp=a186d13400d4af51fbbbe24e855fbfed34a364b21c23d618633ca21c83207ce059&hu=-1&de=1631805339.1631805339.1631805339.1&ce=7cd523d6f7974cf766e24339e7cbdbea` |
| +233 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/jpb.gif?rdm=177085931&qtcurl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&jsuid=pmy0kmnj5ugqkwpd65fogt02&qtsid=1631805339,1631805339,1631805339,1&ppuid=&nu=1&platform=11&fcode=&ffcode=&coop=&weid=8e7ecfe77f644b5ffcbc523be7d2cfad&pru=&fvcode=&mod=intl&flshuid=8649f23e12dc63353be5d879896bc9db&pcau=0&dfp=a186d13400d4af51fbbbe24e855fbfed34a364b21c23d618633ca21c83207ce059` |
| +232 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=0&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=undefined&rn=0.9410525534544941&block=download_float&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=undefined&mod=intl&lang=en_us&stime=1631805339020&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A39%20GMT&stype=` |
| +232 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=0&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=undefined&rn=0.4637524209274938&block=focus_banner_1&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=undefined&mod=intl&lang=en_us&stime=1631805338490&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A38%20GMT&stype=` |
| +232 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=24&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=undefined&rn=0.653032131866931&block=R%3A19518380412&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=undefined&mod=intl&lang=en_us&stime=1631805338443&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A38%20GMT&stype=2&bkt=rotate_2&e=210916d33bff649c3d40e49740d2d3b09d294e&r_area=1&position=0&itemlist=7442193922879401%2C1040910458233801%2C245234401%2C8444901232504101%2C3999298104550701%2C5839466559174501&c1=2%2C2%2C2%2C2%2C2%2C2&rank=0%2C1%2C2%2C3%2C4%2C5&ht=0%2C0%2C0%2C0%2C0%2C0&r_source=%2C%2C%2C%2C%2C&reasonid=%2C%2C%2C%2C%2C` |
| +232 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=0&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=undefined&rn=0.4565153617047062&block=header&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=undefined&mod=intl&lang=en_us&stime=1631805338429&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A38%20GMT&stype=` |
| +232 |  |  | `www.iq.com` | 3 | image | `https://msg-intl.qy.net/act?t=21&bstp=2&p1=1_10_222&u=8649f23e12dc63353be5d879896bc9db&pu=&jsuid=undefined&rn=0.8657938721780778&block=search_box&rpage=home&purl=https%3A%2F%2Fwww.iq.com%2F&rfr=https%3A%2F%2Fwww.iqiyi.com%2Fv_1en7t3axhk4.html%3Fvfrm%3Dpcw_home%26vfrmblk%3DO%26vfrmrst%3D712211_dianshiju_image1&lrfr=undefined&mod=intl&lang=en_us&stime=1631805338360&timezone=Thu%2C%2016%20Sep%202021%2015%3A15%3A38%20GMT&stype=&s2=&s3=&s4=` |
</details>